policies/module-types/commands/src/lib.rs

```diff
@@ -319,7 +319,8 @@ impl Commands {
             "status": status,
             "stdout": stdout,
             "stderr": stderr,
-            "running_time": start_time.elapsed().as_secs(),
+            "running_time": format!("{}.{}", start_time.elapsed().as_secs(),
```

Comment:
* The `elapsed` function actually takes the difference with current time. As we call it twice, the results will be slightly different, and innacurate (espacially around second borders, i.e. 0.999 ans 1.000). We should store the duration once, and format the result from there.
* `subsec_millis` returns a number of milliseconds, so you can't just format it like this. For example a `1.005s` duration will be displayed `1.5`. We could enforce three digits in the format string.
* We should also add a unit to the output value (`1.34s`)